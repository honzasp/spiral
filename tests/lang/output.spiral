(program
  (println 42)
  (println (- 0 21))
  (println 0)
  (println true)
  (println false))
