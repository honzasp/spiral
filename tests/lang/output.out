42
-21
0
true
false
